orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/persistence/ExecutionRepository.java

```diff
@@ -61,15 +64,18 @@ void resume(
   void updateStatus(ExecutionType type, @Nonnull String id, @Nonnull ExecutionStatus status);
 
   @Nonnull
+  @Instrumented(metricName = "retrieveById")
```

Comment:
small change here, this used to be named something not-very-descriptive like `retrieve2`. I favored more descriptive names

Reply:
:thankyou: